### PR TITLE
Todos 3

### PR DESCRIPTION
**Pull request by joeariasc**
- create a model for the todos table

- call the DB connection pool in the Index handler — tx := c.Value(“tx”).(*pop.Connection)

- Query for all of the existing todos — using pop the conventions — hint: tx.All(&todos)

### CODE REVIEW

File: app/actions/todos.go [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
 )
 
 func Index(c buffalo.Context) error {
+	tx := c.Value("tx").(*pop.Connection)
+	todos := models.Todos{}
+	err := tx.All(&todos)
+	if err != nil {
+		return err
+	}
+	todosType := fmt.Sprintf("%T", todos)
```
Comment (danoviedo91) [inline app/actions/todos.go]: Let's remove lines 19 to 24.

File: app/actions/todos.go [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
 	"github.com/gobuffalo/buffalo"
+	"github.com/gobuffalo/pop/v5"
 )
 
 func Index(c buffalo.Context) error {
+	tx := c.Value("tx").(*pop.Connection)
+	todos := models.Todos{}
+	err := tx.All(&todos)
+	if err != nil {
+		return err
```
Comment (danoviedo91) [inline app/actions/todos.go]: returning naked errors are hard to track when the app fails at this point.
You can use this instead, e.g
`c.Error(http.StatusInternalServerError, errors.Wrap(err, "Index - Error while getting all todos"))`
this same error management should be done for the rest of the app.

**Review by danoviedo91 — CHANGES_REQUESTED** [review]

File: app/actions/todos.go (hunk 1 above)
Comment (joeariasc) [reply app/actions/todos.go]: Done!

File: app/actions/todos.go (hunk 2 above)
Comment (joeariasc) [reply app/actions/todos.go]: Done!

**Review by danoviedo91 — APPROVED** [review]

**Pull request merged**